integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/command/SICreationCommand.scala

```diff
@@ -390,8 +391,15 @@ private[sql] case class CarbonCreateSecondaryIndexCommand(
       }
       val indexTablePath = CarbonTablePath
         .getMetadataPath(tableInfo.getOrCreateAbsoluteTableIdentifier.getTablePath)
+      val metaPathMainTbl: String = carbonTable.getMetadataPath
```

Comment:
If metaPathMainTbl and  metaPathSI variable used in one place only, can replace with  `SegmentStatusManager.readLoadMetadata(carbonTable.getMetadataPath)`

Reply:
OK, fixed